### PR TITLE
Change Commit and Freshmaker icons

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
This PR changes icons for the DistGitCommit artifact and the Freshmaker artifact. We use patternfly icons for the rest of them currently.

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
freshmaker_git_commit_icon_change

**Comment by yashvardhannanavati** [comment]
Please wait on merging this, I just realized that the freshmaker and commit icon doesn't turn blue when selected, I will need @mprahl's help on this

File: src/app/story/storyrow/storyrow.component.ts [hunk 1]
```diff
@@ -55,4 +55,21 @@ export class StoryRowComponent implements AfterViewInit {
         return this.node.id;
     }
   }
+
+  useSVG(): boolean {
+    switch (this.node.resource_type) {
+      case('FreshmakerEvent'):
+      case('DistGitCommit'):
+        return true;
```
Comment (mprahl) [inline src/app/story/storyrow/storyrow.component.ts]: You should set a default that returns `false`.

File: src/app/story/storyrow/storyrow.component.html [hunk 2]
```diff
@@ -3,13 +3,24 @@
   <div id="{{ 'js-' + node.resource_type.toLowerCase() + '-node' }}" class="node-column__node" [ngClass]="{'node-column__node--active': active}" 
       [routerLink]="['/', node.resource_type.toLowerCase(), getNodeUid()]"
       tooltip="{{ node.resource_type | nodeTypeDisplay }}: {{ node | nodeUidDisplay }}">
-    <img class="node-column__node-img" src="assets/circle_single.svg"/>
-    <div class="node-column__node-icon"><i class="fa fa-th {{ getNodeIconClass() }}" aria-hidden="true"></i></div>
+    <img class="node-column__node-img" src="assets/circle_single.svg" />
+    <div *ngIf="useSVG();else usePatternfly" class="node-column__node-icon">
+      <svg-icon src="assets/{{ getSVGName() }}" [svgStyle]="{ 'width.px':40 }" aria-hidden="true"></svg-icon>
```
Comment (mprahl) [inline src/app/story/storyrow/storyrow.component.html]: Can `[svgStyle]="{ 'width.px':40 }"` be replaced with CSS in `storyrow.component.css`?

**Comment by mprahl** [comment]
@yashvardhannanavati for the color change, SVG should have a CSS property of `fill` that you can use.

**Comment by mprahl** [comment]
@yashvardhannanavati if you need help fixing the tests, let me know.

**Comment by mprahl** [comment]
@yashvardhannanavati do you actually need the AngularSvgIconModule or could you just use SVG objects directly like in:
https://css-tricks.com/using-svg/#article-header-id-11

**Comment by mprahl** [comment]
Beautiful! :+1:

**Pull request merged**